Submitted By:     Nagy Bence <nagybence@tipogral.hu>
Date:             2005.02.03
Initial Packages: 2.9.5
Description:      This patch adds a new feature to makepkg.
                  If available, use Frugalware's extensinons to makepkg.
                  (provided by the pacman-tools package)

diff -Naur pacman-2.9.5.orig/scripts/makepkg pacman-2.9.5/scripts/makepkg
--- pacman-2.9.5.orig/scripts/makepkg	2004-12-19 03:09:59.000000000 +0100
+++ pacman-2.9.5/scripts/makepkg	2005-02-03 14:43:26.341361040 +0100
@@ -25,6 +25,9 @@
 PKGDEST=$startdir
 USE_COLOR="n"
 
+# Frugalware extensions to makepkg
+[ -e /usr/lib/frugalware/fwmakepkg ] && . /usr/lib/frugalware/fwmakepkg
+
 # source Arch's abs.conf if it's present
 [ -f /etc/abs/abs.conf ] && source /etc/abs/abs.conf
 
@@ -301,7 +304,7 @@
 fi
 
 unset pkgname pkgver pkgrel pkgdesc url license groups provides md5sums force
-unset replaces depends conflicts backup source install build makedepends
+unset replaces depends conflicts backup source install makedepends
 umask 0022
 
 if [ ! -f $BUILDSCRIPT ]; then
